tests/config.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- hosts: all
+
+  vars:
+    manala_sensu_config:
+      EMBEDDED_RUBY: true
```

Comment:
Don't we use the (fucking) macro there ?
"EMBEDDED_RUBY: true" -> "- EMBEDDED_RUBY: true"